packages/icons/packages/AppDownload/AppStore/sprite/index.js

```diff
@@ -9,9 +9,9 @@ if (!process.env.SSR && !process.env.CI) {
   require('./AppStore.svg');
 }
 
-let importPrefix = '';
+let importPrefix = '/sprite.svg';
```

Comment:
Did you test this with CSR start command to see if it works correctly? cuz I think this will break it